zim/gui/widgets.py

```diff
@@ -1998,7 +1998,7 @@ def update_completion(self):
 
 		text = self.get_text()
 		if self._current_completion:
-			if text.startswith(self._current_completion) \
```

Comment:
Ha ha... yeah... I stared at that one cross-eyed for a while too... I think that the variable is serving two purposes. When **one** match is found, it is actually **longer** than the accepted text (so typing semi-clobbers it as you go). Either that, or it is just guarding an unintuitive break condition.

I noticed this after I tried to ignore one-character searches. With the original logic, it would try once, and never again (because "TestPage" **always** starts with "T"); with this swap.... it "works as expected".